avalon/maya/pipeline.py

```diff
@@ -500,11 +500,12 @@ class Creator(api.Creator):
     def process(self):
         nodes = list()
 
-        if (self.options or {}).get("useSelection"):
-            nodes = cmds.ls(selection=True)
+        with lib.undo_chunk():
```

Comment:
Ensure undoing in Maya actually undoes the full instance creation instead of the attributes separately from the node creation, etc. - this fixes undoing potentially leaving "incorrect" instances around that don't have all attributes.